list.go

```diff
@@ -401,3 +404,127 @@ func (db *RoseDB) listSequence(headSeq, tailSeq uint32, index int) (uint32, erro
 	}
 	return seq, nil
 }
+
+// LRem removes the first count occurrences of elements equal to element from the list stored at key.
+// The count argument influences the operation in the following ways:
+// count > 0: Remove elements equal to element moving from head to tail.
+// count < 0: Remove elements equal to element moving from tail to head.
+// count = 0: Remove all elements equal to element.
+// Note that key does not exist, the command will always return 0.
```

Comment:
This method will rewrite the values, so it maybe very slow, we can add a comment to explain it.